BackendIntegrationTests/BackendIntegrationTests.swift

```diff
@@ -13,11 +13,11 @@ import StoreKitTest
 
 class TestPurchaseDelegate: NSObject, PurchasesDelegate {
     var maybeCustomerInfo: CustomerInfo?
-    var consumerInfoUpdateCount = 0
+    var costumerInfoUpdateCount = 0
```

Comment:
s/costumer/customer

Reply:
🤦

Reply:
In my defense this isn't my native language to begin with lol

Reply:
At least I did get the commit message right lol